content/blog/introducing-pirmin-and-laura.md

```diff
@@ -0,0 +1,119 @@
+security; for efficiency (code and hardware designs can be reused, rather than
+reinventing the wheel), and for flexibility. With [Moore's law
+slowing down](https://semiengineering.com/the-impact-of-moores-law-ending/), new
+processors will be more specialised, rather than just smaller and faster.
+Instead, we'll be designing silicon for more specific applications, and ensuring
+the designs are efficient and verifiable. Open source hardware makes this much
+easier - you can get more people working on a design to check it, and you can
+bolt together open modules for different bits of functionality knowing you
+understand what is in them (which you can't do with a proprietary processor core
+which you've licensed - it's just a black box). So open hardware at the silicon
+level is going to be important for the future of computing.
+
+lowRISC itself is still quite small (but growing), but the team size doesn't
+reflect the range of people I’m working with day to day, because there's lots of
+collaboration going on. Open source ecosystems have different kinds of
+organisation and activity in them; lowRISC is focussing on providing quality
+engineering resource and being a hub for collaborative engineering across other
+partners. (A bit like [Linaro](https://www.linaro.org/) does for open source for
+Arm, whose work I’ve always been fascinated by.)  It's a community interest
+company, meaning a nonprofit dedicated to serving a broader mission not just
+itself. I'm looking forward to being part of changing how hardware is developed,
+making it fundamentally more collaborative and factoring in testing and
+maintenance sensibly. 
+
+There are some really interesting challenges. For instance, you can't be 100%
+open right now in silicon, because in an actual manufactured chip there's a lot
+of analogue components as well as the open source digital bits, and those
+analogue bits are generally closely linked to the big foundries which
+manufacture silicon and are secret sauce today.  Collaborative engineering
+should be cost effective and useful to companies, as a way of working on
+non-differentiating technology components, but it's not always an easy sell or
+easy to make happen in practice. The same goes for opening up hardware IP - not
+always a straightforward case to make. How should the governance for this sort
+of work operate, and how do you make it work in practice? And of course we have
+the usual startup challenge, that we need to make reliable, high quality
+products, in a reasonable time, which people actually want and are willing to
+pay for.
+
+I’m really looking forward to seeing what we can accomplish, and being part of
+the move towards open source silicon.
+
+(If you’d like to learn more about how I ended up here, I wrote on [my personal
+blog](https://lbj20.blogspot.com/2019/04/new-challenges-ahead.html) about this
+role and how it fits with things I’ve been working on.)"
+
```

Comment:
Maybe add `----` here so a separator is generated

Reply:
Done.